.github/workflows/pull-request.yml

```
@@ -0,0 +1,19 @@
+name: Run build tests
+
+'on':  
+  push:
+  pull_request:
+    branches:
+      - main
+
+jobs:  
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dependencies
+      run: npm install
+    - name: Generate .env file
+      run: cp .env.example .env
+    - name: Build
+      run: npm run build
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Add two build commands

```suggestion
      run: |
        npm run build
        npm run buildScript
```

---

Added two build commands